src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -77,25 +77,18 @@ void OnSystemRequestNotification::Run() {
     return;
   }
 
+  const std::string& filename =
```

Comment:
@LitvinenkoIra 
My concern is that for request type other than PROPRIETARY or HTTP you'll be creating an empty fields 'file_name' and 'binary_data' so I think its better to add check for file name existence and thus binary data appending.

Reply:
@dev-gh Fixed af1e068d46489e1efd50644020c80d280dd2e737